native/components/message-markdown-rules.react.js

```diff
@@ -0,0 +1,155 @@
+// @flow
+
+import * as React from 'react';
+import { Text } from 'react-native';
+import SimpleMarkdown from 'simple-markdown';
+
+type State = { [string]: any };
```

Comment:
There are a lot of `any` types here. In the case where `simple-markdown` types it as `any`, it's okay to use. But let's try to avoid it as much as possible.

Also - if any of these types are possibly to import directly from `simple-markdown` or a file within it, that would be preferable to redefining them.